Sasquatch/SasquatchObjC/Info.plist

```diff
@@ -41,5 +41,18 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>None</string>
+			<key>CFBundleURLName</key>
+			<string>com.microsoft.azure.mobile.sasquatchobjc</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>mobilecenter-3ccfe7f5-ec01-4de5-883c-f563bbbe147a</string>
```

Comment:
This one should be mobilecenter-0dbca56b-b9ae-4d53-856a-7c2856137d85 to match the App key from the start call in the app delegate.